mirage/config.js

```diff
@@ -0,0 +1,31 @@
+export default function () {
+  // These comments are here to help you get started. Feel free to delete them.
+  /*
+    Config (with defaults).
+
+    Note: these only affect routes defined *after* them!
+  */
+  // this.urlPrefix = '';    // make this `http://localhost:8080`, for example, if your API is on a different server
+  // this.namespace = '';    // make this `/api`, for example, if your API is namespaced
+  // this.timing = 400;      // delay for each request, automatically set to 0 during testing
+  /*
+    Shorthand cheatsheet:
+
+    this.get('/posts');
+    this.post('/posts');
+    this.get('/posts/:id');
+    this.put('/posts/:id'); // or this.patch
+    this.del('/posts/:id');
+*/
+  this.get('/users');
+  this.get('/users/:id');
+  this.post('/users');
+  this.put('/users/:id');
+  this.get('/gameHistories');
+  this.get('/gameHistories/:id');
+  this.post('/gameHistories');
+  this.put('/gameHistories/:id');
```

Comment:
For better readability please split by empty line difference endpoints:
```suggestion
  this.get('/users');
  this.get('/users/:id');
  this.post('/users');
  this.put('/users/:id');
  
  this.get('/gameHistories');
  this.get('/gameHistories/:id');
  this.post('/gameHistories');
  this.put('/gameHistories/:id');
```